ci/build-helpers.sh

```diff
@@ -19,22 +19,22 @@ function report_state () {
 
   # Push to InfluxDB if configured
   if [ -n "$INFLUXDB_WRITE_URL" ]; then
-    printf 'prcheck,checkname=%s host="%s",state="%s",cihash="%s",uptime=%s%s %s' \
-           "$CHECK_NAME/$WORKER_INDEX" "$(hostname -s)" "$current_state" "$CI_HASH" $((time_now - TIME_STARTED)) \
-           "${prtime:+,prtime=$prtime}${LAST_PR:+,prid=\"$LAST_PR\"}${LAST_PR_OK:+,prok=$LAST_PR_OK}" $((time_now * 1000000000)) |
+    printf 'prcheck,checkname=%s host="%s",state="%s",prid="%s"%s %s'                 \
+           "$CHECK_NAME/$WORKER_INDEX" "$(hostname -s)" "$current_state" "$PR_NUMBER" \
+           "${prtime:+,prtime=$prtime}${PR_OK:+,prok=$PR_OK}" $((time_now * 10**9))   |
       case "$INFLUXDB_WRITE_URL" in
         # If INFLUXDB_WRITE_URL starts with insecure_https://, then strip
         # "insecure_" and send the --insecure/-k option to curl.
         insecure_*)
-          curl --max-time 20 -XPOST "${INFLUXDB_WRITE_URL#insecure_}" -k --data-binary @- || true;;
```

Comment:
Are you not exiting on error anymore?

Reply:
I moved the `|| true` after the `esac`.

But the CI builder hasn't been run with `bash -e` since I've been here, and it's not in a state to work with that at the moment. I could make it compatible, but it would take a bit of work.